include/core/mobility_cahn_hilliard_model.h

```diff
@@ -20,57 +20,66 @@
 #include <core/interface_property_model.h>
 
 /**
- * @brief Abstract class that allows to calculate the
- * mobility for the Cahn-Hilliard-Navier-Stokes equations.
+ * @brief Implementation of a general case computation of the mobility
```

Comment:
What do you mean by general case here? It's a bit hard to understand.

Reply:
There exist different models to compute the mobility, that is what I mean.

Reply:
Actually I'll just remove the "general case", it is not required